### PR TITLE
Add P33, P99, not P167 space

### PR DESCRIPTION
**Pull request by StevenClontz**
Shown by KP here: https://mathoverflow.net/questions/452903/must-us-extremally-disconnected-spaces-be-sequentially-discrete?noredirect=1#comment1171878_452903

### CODE REVIEW

**Comment by StevenClontz** [comment]
converted back to draft - I think this space is KC.

File: spaces/S000020/README.md [hunk 1]
```diff
@@ -14,6 +15,7 @@ Let $X=\omega\cup\{\infty\}=\{0,1,2\dots\}\cup\{\infty\}$.
 Define $U \subseteq X$ to be open if its complement either is finite or includes $\infty$.
 
 This space is the one-point compactification of a countably infinite discrete space.
+Equivalently, it is homeomorphic to $\{0\}\cup\{2^{-n}:n<\omega\}$ as a subspace of {S25}.
```
Comment (prabau) [inline spaces/S000020/README.md:18]: No need for the word "Equivalently", I think.  Also, maybe slightly simpler: replace $2^{-n}$ with $1/n$.

File: spaces/S000183/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+uid: S000183
+name: Non-sequentially discrete countable complement topology
+aliases:
+  - Modified cocountable space
```
Comment (prabau) [inline spaces/S000183/README.md:5]: Hmm, how about removing the alias and merging it into the name, something like: 
"Non-sequentially discrete modified cocountable topology"?

File: spaces/S000183/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+refs:
+  - mo: 3198478
```
Comment (prabau) [inline spaces/S000183/README.md:7]: The mo id should be 452903.  And same thing in the other files.

**Comment by prabau** [comment]
P80([Fréchet Urysohn) = false.  This can be replaced with the stronger P81(countably tight) = false.

File: spaces/S000020/README.md (hunk 1 above)
Comment (StevenClontz) [reply spaces/S000020/README.md:18]: I'm scared of division by zero so I prefer to use 2^{-n}.

File: spaces/S000183/README.md (hunk 2 above)
Comment (StevenClontz) [reply spaces/S000183/README.md:5]: I'll make a modification for this and https://topology.pi-base.org/spaces/S000017 to line them up a bit.

Comment (prabau) [reply spaces/S000183/README.md]: I like the new name for S17 too.

File: spaces/S000020/README.md (hunk 1 above)
Comment (prabau) [reply spaces/S000020/README.md]: Well, one could have specified $n=1,2,\dots$, but ok.

Comment (StevenClontz) [reply spaces/S000020/README.md]: It's a silly hill, and not quite one I'm willing to die on, but thanks for your understanding. 😆

**Comment by prabau** [comment]
For P103 (strongly KC), it must be something simple, but one part of the argument escapes me:
Given A = countably compact subset of the space.  Why is $A\cap Y$ countably compact (with $Y$ = the convergent sequence)?
(this is then used to infer that $A\cap Y$ is closed in $Y$)

**Comment by StevenClontz** [comment]
> For P103 (strongly KC), it must be something simple, but one part of the argument escapes me: Given A = countably compact subset of the space. Why is A∩Y countably compact (with Y = the convergent sequence)? (this is then used to infer that A∩Y is closed in Y)

Since $Y$ is closed, $A\cap Y$ is a closed-in-$X\cup Y$ subset of a countably compact $A$, and thus is countably compact. This remains true when considering $A$ has a subset of $Y$.

**Comment by prabau** [comment]
Another not very important comment, but I find the reader has to make an effort to keep things straight.  We usually use $X$ for the whole space, and it would be useful to give it the name of $X$ and call the two parts $Y$ and $Z$.  Then we can easily refer to the whole thing as $X$ where needed.  (If you think it's too much work but are not against the idea, I can give it a try.)

**Comment by StevenClontz** [comment]
I'll make that change. I actually started just implicitly letting X be the whole space when defining a few properties in my other PR, so that's a good convention to use.

**Comment by prabau** [comment]
Is the space really the minimal topology on the disjoint union that makes the convergent sequence closed in the whole space?

Considering this in general, suppose we have two topological spaces $Y$ and $Z$, disjoint as sets, and we want to give $X=Y\cup Z$ the smallest topology $\tau$ (if there is one) such that the subspace topologies induced on $Y$ and $Z$ are their original topologies and $Z$ is closed in $X$.

First of all, since $Z$ is closed in $X$, $Y$ must be open in $X$ and all the opens in $Y$ are in $\tau$.  Also, all the closed sets in $Z$ are closed in $X$, so $\tau$ contains all the complements in $X$ of the closed sets in $Z$.  And it seems that these two types of open sets combined already gives a topology, which is necessarily equal to $\tau$.  In particular, any neighborhood in $X$ of a point in $Z$ would have all of $Y$ as a subset.

If the above is correct, that is a smaller topology than what is described in the PR.

**Comment by prabau** [comment]
Elaborating on the construction in my previous comment, it seems that the space in this PR is more of a "blowup" kind of construction.  (Not sure if there is a better name for this, but someone must have done something similar somewhere.)

One takes a space $Y$ (cocountable topology on the reals in this case) and a closed point $y_0\in Y$.  One then replace $y_0$ with a (closed) topological subspace $Z$ (convergent sequence here).  Nbhds of $z\in Z$ are obtained from nbhds of $y_0$ in $Y$ by replacing $y_0$ in it with some nbhd of $z$ in $Z$.  And a local base of nbhds of $y\in Y$ with $y\ne y_0$ remain the same within $Y\setminus\\{y_0\\}$.

**Comment by StevenClontz** [comment]
> any neighborhood of a point of $Z$ would have all of $Y$ as a subset

This wouldn't be a topology as it would violate the finite intersection property. Once you ensure finite intersections of open are open, I think we have exactly what we need.

>  it seems that the space in this PR is more of a "blowup" kind of construction.

That idea occurred to me as well, but since I think my minimal topology definition works, it seemed to be the slickest method.

**Comment by prabau** [comment]
> > any neighborhood of a point of Z would have all of Y as a subset
> 
> This wouldn't be a topology as it would violate the finite intersection property. Once you ensure finite intersections of open are open, I think we have exactly what we need.

No, it does not violate the intersection property.  We are talking here about the axioms for a topology in terms of nbhds.  Given a point $x\in Z$ and two nbhds of $z$ in $X$, their intersection is still a nbhd of $z$ (and all such nbhds contain $Y$).

On the other hand, using the axioms for a topology in terms of open sets, there is no problem either.  Suppose $U_1$ and $U_2$ are two open sets in $X$ with $U_1$ intersecting $Z$ and $U_2\subseteq Y$.  Then $U_1$ necessarily contains $Y$, so $U_1\cap U_2=U_2$, which is open in $X$.

**Comment by StevenClontz** [comment]
Thanks, you're correct. Even saying minimal $T_1$ topology wouldn't be sufficient here as then neighborhoods of points in $Z$ would need only be cofinite in $Y$, not cocountable. (Maybe that does the trick as well, but it's not the original example.) I'll make a correction.

**Comment by StevenClontz** [comment]
In particular, the minimal $T_1$ topology would not be US: every point in $Z$ would be a limit of any sequence in $Y$, as countable subsets of $Y$ are no longer closed.

**Comment by prabau** [comment]
> In particular, the minimal T1 topology would not be US: every point in Z would be a limit of any sequence in Y, as countable subsets of Y are no longer closed.

Yeah, good point.

**Comment by prabau** [comment]
Your latest description is a good one.  Taking one last look at the whole thing.

**Comment by prabau** [comment]
Changed a few minor things.  It's ready to go.  Let me know if you are ok with it.

**Comment by StevenClontz** [comment]
Looks good

**Review by prabau — APPROVED** [review]

**Pull request merged**